website_and_docs/content/documentation/webdriver/elements/interactions.en.md

```diff
@@ -54,98 +110,79 @@ possible keystrokes that WebDriver Supports.
 
 {{< tabpane langEqualsHeader=true >}}
   {{< tab header="Java" >}}
-import org.openqa.selenium.By;
-import org.openqa.selenium.Keys;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.firefox.FirefoxDriver;
-
-public class HelloSelenium {
-  public static void main(String[] args) {
-    WebDriver driver = new FirefoxDriver();
-    try {
       // Navigate to Url
-      driver.get("https://google.com");
-
-      // Enter text "q" and perform keyboard action "Enter"
-      driver.findElement(By.name("q")).sendKeys("q" + Keys.ENTER);
-    } finally {
-      driver.quit();
-    }
-  }
-}
+      driver.get("https://www.selenium.dev/selenium/web/inputs.html");
+
+      // Clear field to empty it from any previous data
+      driver.findElement(By.name("email_input")).clear();
+	  
+	  //Enter Text
+	  driver.findElement(By.name("email_input")).sendKeys("admin@localhost.dev");
+    
 
   {{< /tab >}}
   {{< tab header="Python" >}}
-from selenium import webdriver
-from selenium.webdriver.common.by import By
-from selenium.webdriver.common.keys import Keys
-driver = webdriver.Firefox()
+
 
     # Navigate to url
-driver.get("http://www.google.com")
+	driver.get("https://www.selenium.dev/selenium/web/inputs.html")
+
+    # Clear field to empty it from any previous data
+	driver.find_element(By.NAME, "email_input").clear()
+
+	# Enter Text
+	driver.find_element(By.NAME, "email_input").send_keys("admin@localhost.dev" )
 
-    # Enter "webdriver" text and perform "ENTER" keyboard action
-driver.find_element(By.NAME, "q").send_keys("webdriver" + Keys.ENTER)
   {{< /tab >}}
   {{< tab header="CSharp" >}}
-using (var driver = new FirefoxDriver())
-{
+
   // Navigate to Url
-  driver.Navigate().GoToUrl("https://google.com");
+  driver.Navigate().GoToUrl("https://www.selenium.dev/selenium/web/inputs.html");
 
-  // Enter "webdriver" text and perform "ENTER" keyboard action
-  driver.FindElement(By.Name("q")).SendKeys("webdriver" + Keys.Enter);
+  // Clear field to empty it from any previous data
+  driver.FindElement(By.Name("email_input")).Clear();
+  
+  //Enter Text
+  driver.FindElement(By.Name("email_input")).SendKeys("admin@localhost.dev");
+  
+  
 }
   {{< /tab >}}
   {{< tab header="Ruby" >}}
-require 'selenium-webdriver'
-driver = Selenium::WebDriver.for :firefox
-begin
+
     # Navigate to URL
-  driver.get 'https://google.com'
+	driver.get 'https://www.selenium.dev/selenium/web/inputs.html'
 
-    # Enter "webdriver" text and perform "ENTER" keyboard action
-  driver.find_element(name: 'q').send_keys 'webdriver', :return
+    # Clear field to empty it from any previous data
+	driver.find_element(name: 'email_input').clear
+	
+	# Enter Text
+	driver.find_element(name: 'email_input').send_keys 'admin@localhost.dev'
 
-ensure
-  driver.quit
-end
   {{< /tab >}}
   {{< tab header="JavaScript" >}}
-const {Builder, By, Key} = require('selenium-webdriver');
 
-(async function example() {
```

Comment:
PR Looks good to me!

But the main idea is to move code samples to examples folder and render it from there!

Reply:
i don't know what that means. explain that, and i will then make the changes there. @harsha509

Reply:
Hi @harsha509 , can we accept this PR, as there are two components of change with respect to documentation - 1 is to update examples to the web pages listed on selenium.dev and another is to add in code files. let me know. thanks.